test/unit/foundation.test.ts

```diff
@@ -131,4 +146,119 @@ describe('foundation', () => {
     it('does not render empty objects into its template', () =>
       expect(empty).dom.to.be.empty);
   });
+
+  describe('isSame', () => {
+    it('is true of any two SCL Elements', () => {
+      expect(isSame(scl1, scl2)).to.be.true;
+    });
+
+    it('is true of any two Header Elements', () => {
+      expect(
+        isSame(scl1.querySelector('Header')!, scl2.querySelector('Header')!)
+      ).to.be.true;
+    });
+
+    it('is true of any two Communication Elements', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
    it("is true of any two Communication Elements", () => {
```